.travis.yml

```diff
@@ -57,12 +57,15 @@ matrix:
         - go get github.com/ckaznocha/protoc-gen-lint
         - go get golang.org/x/tools/go/packages
         - cp go.mod.backup go.mod
+        # install protolock
+        - go get -u github.com/nilslice/protolock/...
```

Comment:
Move it upwards before restoring go.mod, and don't use `go get -u` it will try to reupdate everything that has already been cleanly installed.

Reply:
and use the concerete path of `github.com/nilslice/protolock/cmd/protolock`, might avoid downloading some dependencies